pyincore/models/fragilitycurveset.py

```diff
@@ -142,6 +147,7 @@ def calculate_damage_interval(
             ("earthquake+tsunami", "building", 3): FragilityCurveSet._3ls_to_4ds,
             ("tornado", "building", 3): FragilityCurveSet._3ls_to_4ds,
             ("hurricane", "building", 3): FragilityCurveSet._3ls_to_4ds,
+            ("hurricane", "warehouse", 3): FragilityCurveSet._3ls_to_4ds,
```

Comment:
maybe this can be reused using the building damage; since warehouse is just building.